agent/cmd/cuda-vmm-interpose/Makefile

```
@@ -0,0 +1,35 @@
+COORDINATOR_SOURCES := coordinator.c util.c
+COORDINATOR_HEADERS := protocol.h util.h
+COMMON_FLAGS := -std=gnu11 -O2 -Wall -Wextra -Werror
+
+.PHONY: all clean
+
+all: $(INTERPOSER) $(COORDINATOR)
+
+$(BUILD_DIR):
+	mkdir -p $@
+
+$(INTERPOSER): $(INTERPOSER_SOURCES) $(INTERPOSER_HEADERS) | $(BUILD_DIR)
```

checkmake (0.3.2)

[warning] Target body for "$(INTERPOSER)" exceeds allowed length of 5 lines (6).

(maxbodylength)